src/core2/graph.js

```diff
@@ -433,10 +438,51 @@ class InternalReference implements NodeReference {
     return this._graph._nodes[indexDatum.index].node;
   }
 
-  neighbors(
+  *neighbors(
     options?: NeighborsOptions
   ): Iterator<{|+ref: NodeReference, +edge: Edge<any>|}> {
-    const _ = options;
-    throw new Error("Not implemented");
+    const indexDatum = this._graph._nodeIndices.get(this._address);
+    if (indexDatum == null) {
+      return;
+    }
+    const nodeIndex = indexDatum.index;
+
+    const direction = (options && options.direction) || "ANY";
```

Comment:
Done

Reply:
Update: Doing this gave me a typing headache, around properly expressing the set `{||} | {|+plugin|} | {|+plugin, +type|}`. Once I did that, it wasn't happy to assign the `{}` as a PluginFilter in the default neighbor options above.
I'm going to merge it as is, since the code is fine as written. Feel free to PR an improvement!

Reply:
Yep; that known issue is why I didn’t suggest it outright. I spent a bit
of time poking around in Flow’s inference engine to see if there might
be an easy way to infer `{}` as having either type `{||}` or the
unsealed object type, but didn’t find much useful. It wouldn’t be as
simple as inferring the intersection type `{} & {||}`; this would be
unsound. One interesting thing that I learned is that `flow type-at-pos`
in fact infers `{||}` as the type of `const emptyObject = {}`, which is
surprising given that `const emptyObject: {||} = {}` yields a type
error. But that’s not particularly actionable, as far as I can tell.

So, TL;DR: merging as-is is fine with me.